source/docs/software/commandbased/organizing-command-based.rst

```diff
@@ -203,7 +203,21 @@ Instance factory methods work great for single-subsystem commands.  However, com
 
   .. code-block:: c++
 
-    // TODO
+    class AutoRoutines {
```

Comment:
In C++ these can and should be free functions in a namespace, rather than a class.